doc/user/layouts/shortcodes/session-variables.html

```diff
@@ -1,6 +1,6 @@
 ## Session variables
 
-Name                                        | Default value             |  Description                                                          | [Settable](../alter-system-set)
+Name                                        | Default value             |  Description                                                          | [Can Alter System Default](../alter-system-set)
```

Comment:
Could we leave this as `Settable`? It'd be more worth it to add a short blurb before the table that explains under which conditions (and roles) these can be set. I can do that as a follow-up.

Reply:
Done!